docs/sources/troubleshooting/support-bundles/index.md

```diff
@@ -69,6 +71,8 @@ server_admin_only = true
 public_keys = ""
 ```
 
+> **Note**: By default, only server admins can generate and view a support bundle. The `server_admin_only` option can be used to widen this access to organization admins. 
```

Comment:
```suggestion
By default, users with the following permissions can generate and view a support bundle:

- In Grafana Cloud, only administrators can generate and view a support bundle. 
- In Grafana on-premises, only server admins can generate and view a support bundle. The `server_admin_only` option can be used to widen this access to organization admins. 
```

Since support bundles are available in Grafana Cloud for admins, I think we can turn this into couple of lines of clarification. The `server_admin_only` and Server Admin availability is applicable only for on-premises.